schema/plugin.json

```
@@ -56,10 +56,25 @@
         "type": "string"
       }
     },
+    "createEmptySection": {
+      "type": "boolean",
+      "title": "Show create empty section",
```

Would it be more readable if we added quotes and capitalization here?

```suggestion
      "title": "Show \"Create Empty\" section",
```